test/detect/angelscript/default.txt

```diff
@@ -53,4 +55,9 @@ void Main()
 {
     SomeClass@ c = SomeClass();
     c.DoSomething();
+    @c = null;
```

Comment:
Another testing change or?

Reply:
Indeed, just noticed that after my last commit, sorry! :sweat_smile: